include/dxc/DXIL/DxilShaderFlags.h

```diff
@@ -204,6 +204,8 @@ class ShaderFlags {
   bool GetWriteableMSAATextures() const { return m_bWriteableMSAATextures; }
 
   // SM 6.8+
+  void setReserved(bool flag) { m_bReserved = flag; }
```

Comment:
Presumably `setReserved` is never / should never be called?  Any reason to keep it?

Reply:
To work around the warning on Linux build for unused private field.

Reply:
Well, that warning is broken because it is still unused :)  

Is there any other way we can avoid the warning that doesn't involve adding dummy code?

Reply:
Changed to diagnostic push/pop.

Reply:
I'm surprised there's no better way to do this!